### ISSUE TITLE
Being able to create backups without including the app's data

### ISSUE THREAD

**Issue by MadMan247** [YunoHost/issues#2408]
When peertube is backed up for an upgrade, there is a line in the console such as this:
/home/yunohost.app/peertube will not be saved, because 'BACKUP_CORE_ONLY' is set.

I notice that this is not a choice when doing a manual backup through the admin UI, but I think that it would be a beneficial addition. I think of myself going in and botching a configuration, I would like for a backup to save me without backing up a ridiculous amount of data that I don't need to.

I imagine a switch or another checkbox on the backup page for each item that supports this, such as file browser, immich, peertube, and other apps which typically have hosted data in addition to configuration files. Apps such as tinytodo would likely not have this option

**Comment by alexAubin** [comment]
Sounds like it should just be peertube's backup/restore scripts that should explicitly backup/restore that file instead of assuming the data dir is always backed up ...

**Comment by MadMan247** [comment]
I fear that you misunderstand my intent; the peertube backup scripts work properly as far as I am concerned. I have not had any issues with this. My only issue is that the backups that I can do manually ***always*** include the data dir, which for peertube is **ridiculously large**. Since, upon an application upgrade, the logic has already been implemented (and shown to work) to only backup the configuration without the data dir, I would like the choice to make a manual backup implementing this, so that my peertube backup for a single config file that I messed up isn't 100+GB.

And don't get me wrong, I want to be able to back up the data dir from time to time, I just want it to be a choice. If I'm digging around peertube's files (e.g. to enable redundancy), and I accidentally delete something, I'd rather use a ~2GB backup, but that doesn't mean I never want to keep a full ~100GB backup.

Basically, I don't think that 'bug' is the proper label. Maybe I'm misunderstanding.

Thank you for your time and attention.

**Comment by alexAubin** [comment]
My bad, I totally read things way too quickly :upside_down_face: 

I guess this is answered by this piece of info kind of burried in the doc : https://yunohost.org/en/backup/include_exclude_files#do-not-backup-large-amoun

You can either run (from the CLI) `BACKUP_CORE_ONLY=1 yunohost backup create --apps nextcloud` (for nextcloud in this case)

or permanently with: `yunohost app setting nextcloud do_not_backup_data -v 1`

But yeah the option ain't available from the webadmin

**Comment by MadMan247** [comment]
I see, I'll keep that in mind. That is functionality that I desired. While my thoughts were to be able to do this in webadmin, in this context I was already going to be on the command line to edit files, I suppose that it makes sense to use the command line to back up files.

I will close this request. Thank you!
